### DISCUSSION TITLE
I am encountering bugsnag error while trying to run the app

### DISCUSSION THREAD

**Discussion by rajoriaakash** [RocketChat/Rocket.Chat.ReactNative#3391] [Q&A] [answered]
**Description:**
Can not run the project after following CONTRIBUTING.md, shows a problem with ':bugsnag_react-native'.

**Steps to reproduce:**
Clone repository by git clone git@github.com:RocketChat/Rocket.Chat.ReactNative.git .
Move to subfolder by cd Rocket.Chat.ReactNative .
Install dependencies with yarn.
4.Run yarn android command.

**Expected behavior:**
The app should get running without any error according to Contribution guide.

**Actual behavior:**
I am having the following error regarding bugsnag :

![image](https://user-images.githubusercontent.com/59030114/133998419-f86d640c-56c2-40f6-8272-ee7afc6cfde8.png)

I opened an issue #3389 on this too.
Can someone please guide me through?Thank You

**Comment by diegolmello** [comment, marked as answer]
Answered on #3389
